daemon/cmld.c

```diff
@@ -920,11 +919,23 @@ cmld_container_start(container_t *container)
 		}
 		time_register_clock_check();
 	} else {
-		audit_log_event(container_get_uuid(container), FSA, CMLD, CONTAINER_MGMT,
-				"container-start-already-running",
-				uuid_string(container_get_uuid(container)), 0);
-		DEBUG("Container %s has been already started",
+		DEBUG("Container %s is not startable. Checking if already running.",
 		      container_get_description(container));
+		// differentiate between a running container (which does not need to be started) and a 'broken' container state, that cannot be started
+		if ((container_get_state(container) == COMPARTMENT_STATE_STARTING) ||
+		    (container_get_state(container) == COMPARTMENT_STATE_RUNNING)) {
+			audit_log_event(container_get_uuid(container), FSA, CMLD, CONTAINER_MGMT,
+					"container-start-already-running",
+					uuid_string(container_get_uuid(container)), 0);
+			DEBUG("Container %s has been already started",
+			      container_get_description(container));
+		} else {
+			audit_log_event(container_get_uuid(container), FSA, CMLD, CONTAINER_MGMT,
+					"container-start-failed",
```

Comment:
do nor introduce a new audit message here. FSA "container-start" is sufficient and would be in accordance with FSP documenteation